src/test/java/org/minions/devfund/reinaldo/movie/MovieDatabaseTest.java

```diff
@@ -0,0 +1,145 @@
+package org.minions.devfund.reinaldo.movie;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by reinaldo on 26-07-18.
+ */
+public class MovieDatabaseTest {
+
+    private static final int FIRST_MOVIE = 0;
+    private static final double PARAMETER_FOR_DOUBLE = 0;
+    public static final String THE_PIANIST2 = "The Pianist2";
+
+    /**
+     * Get best actor in the collection.
+     */
+    @Test
+    public void testGetBestActor() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+
+        final String movie1 = "Monster";
+        final String[] movie1WithActors = {"Denzel Washington"};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Denzel Washington" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)